en/docs/reusable-components/choreo-connectors-list.md

```diff
@@ -0,0 +1,34 @@
+The following lists the connections you can use to interact with generic protocol-based APIs:
+
+| **Connection** | **Description**                                               |
+|----------------|---------------------------------------------------------------|
+| **HTTP**       | Communicates with external endpoints over HTTP.               |
+| **SMTP**       | Performs email-related operations over SMTP.                  |
+| **POP3**       | Retrieves emails from a mail server over POP3.                |
+| **IMAP**       | Retrieves emails from a mail server over a TCP/IP connection. |
+
+The following lists the connections you can use to interact with application-specific APIs:
+
+| **Connection**      | **Description**                                                                                |
+|---------------------|------------------------------------------------------------------------------------------------|
+| **Mail by Choreo**  | Send email via Choreo to a specific address with a specific subject and a message body.        |
+| **SMS by Choreo**   | Send SMS via Choreo to a specific telephone number with a specific message body.               |
+| **COVID-19 API**    | Integrate with the COVID-19 API to retrieve statistics relating to the spread of COVID-19 disease. You can perform operations such as retrieving the number of new cases in a specific country, retrieve details relating to the vaccine coverage in a continent, etc. |
```

Comment:
Why yse `retrieving` and `retrieve` both?

Reply:
Btw, Can we reduce the length may be: 

Integrate with the COVID-19 API to retrieve the statistics related to the spread of  the disease, such as number of new cases in a specific country, vaccine coverage data, etc.

Reply:
Added the suggested test as the description.